command/version/result.go

```diff
@@ -1,9 +1,16 @@
 package version
 
+import "fmt"
+
 type VersionResult struct {
-	Version string `json:"version"`
+	Version   string `json:"version"`
+	Commit    string `json:"commit"`
+	BuildTime string `json:"buildTime"`
 }
 
 func (r *VersionResult) GetOutput() string {
-	return r.Version
+	return fmt.Sprintf("\n[VERSION INFO]\n"+
+		"Release version: %s \n"+
+		"Commit hash: %s\n"+
+		"Build time: %s", r.Version, r.Commit, r.BuildTime)
```

Comment:
How about using `helper.FormatKV` for common format?

Reply:
Completely forgot that we have that 😸 Fixed f3eed73e61d93fbad1f36a453390a7ffc932cd36